### PR TITLE
DIRECTOR: Add Dutch detection entries

### PR DESCRIPTION
**Pull request by lemtom**
Adds new titles to the detection tables, mostly Dutch localizations.

### CODE REVIEW

File: engines/director/detection_tables.h [hunk 1]
```diff
@@ -3480,6 +3532,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("sakuratoasobo", "", "Let\'s Play with Sakura", "8d83064402198a909b0abcebc6b46d2b", 118198, Common::JA_JPN, 602),
 	WINGAME1_l("sakuratoasobo", "", "Let\'s Play with Sakura.exe", "45871c12eb944f09f50ee742113a1e2d", 1982850, Common::JA_JPN, 650),
 
+	WINGAME1_l("samson", "", "start.exe", "a593079aecf5bd938ce75264cac24b2d", 1919963, Common::NL_NLD, 600),
```
Comment (trembyle) [inline engines/director/detection_tables.h:3535]: We should have a second file for this one. `start.exe` is extremely common.

File: engines/director/detection_tables.h [hunk 2]
```diff
@@ -3590,6 +3651,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("fg25ans", "", "FG25Ans", "dec72b4b6f8e97f1ea24361794bf0bdf", 114301, Common::FR_FRA, 702),
 	WINGAME1_l("fg25ans", "", "FG25Ans.exe", "88d4c24f6d4b78a73ce5b07693eddbc8", 2412706, Common::FR_FRA, 702),
 
+	//Miel Monteur - Miel Bouwt Auto's!
+	WINGAME1_l("garygadget1", "", "Start.exe", "c1a2e8b7e41fa204009324a9c7db1030", 2933829, Common::NL_NLD, 700),
```
Comment (trembyle) [inline engines/director/detection_tables.h:3655]: same here

File: engines/director/detection_tables.h [hunk 3]
```diff
@@ -3480,6 +3536,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1_l("sakuratoasobo", "", "Let\'s Play with Sakura", "8d83064402198a909b0abcebc6b46d2b", 118198, Common::JA_JPN, 602),
 	WINGAME1_l("sakuratoasobo", "", "Let\'s Play with Sakura.exe", "45871c12eb944f09f50ee742113a1e2d", 1982850, Common::JA_JPN, 650),
 
+	WINGAME2_l("samson", "", "start.exe", "a593079aecf5bd938ce75264cac24b2d", 1919963,
+								"START.DXR", "ad34b9d4987fc84c238f0e88b174fbc9", 229050, Common::NL_NLD, 600),
```
Comment (sev-) [inline engines/director/detection_tables.h:3540]: Do you have a better file for the second detection target? E.g. something less generic than "Start.dxr"?

File: engines/director/detection_tables.h [hunk 4]
```diff
@@ -1851,6 +1868,12 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1("blenderbest", "", "PPC LAUNCHER", "9d10bbbf1b8d527e203c6be2eac3518c", 58058, 404),
 	WINGAME1("blenderbest", "", "BLENDPC.EXE", "408fdb0f6806d93a9af0412f3704481e", 728849, 404),
 
+	WINGAME1("blinkybill1", "", "BLINKY.EXE", "0d374b4a559e505c0db63cabc2219380", 712501, 404),
+	//Blinky Bill 1 en het verhaal van de Griezel-Grot
+	WINGAME1_l("blinkybill1", "", "start.exe", "be1e65196a1d1c810d84c0084a5c1033", 717299, Common::NL_NLD, 404),
```
Comment (sev-) [inline engines/director/detection_tables.h:1873]: Do you have a second file to throw in here for detection?

File: engines/director/detection_tables.h [hunk 5]
```diff
@@ -3236,6 +3284,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINGAME1_l("bigmax", "nº20", "BIG32.EXE", "518a98696fe1122e08410b0f157f21bf", 7313980, Common::PT_BRA, 602),
```
Comment (sev-) [inline engines/director/detection_tables.h:3285]: You have non-ascii symbol here in the second parameter. If you have to have it, then please replace it with the octal codes

File: engines/director/detection_tables.h [hunk 6]
```diff
@@ -3236,6 +3284,8 @@ static const DirectorGameDescription gameDescriptions[] = {
...
 
+	WINGAME1_l("biomania", "", "start.exe", "518a98696fe1122e08410b0f157f21bf", 1518949, Common::NL_NLD, 600),
```
Comment (sev-) [inline engines/director/detection_tables.h:3287]: We need a second file here, not just "start.exe"

**Comment by sev-** [comment]
Thanks!

**Pull request merged**